app/views/posts/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>post index</h1> <br>
+
+<% @posts.each do |e|%>
+<%= e.title %> 
+<%= e.id %>
+  <% if logged_in? %>
+  <%= e.user.username %>
```

Comment:
If we create a user with `email` and `password` attributes only (without `username`), then this would be `nil` and nothing would be displayed herein. Please note that requirements state only `email` and `password` as requirements to login, but does not mention anything about what fields are required for signing up. However, if you're displaying `username` as author's name of a post, then please make sure that:
1. There's a Sign Up feature
2. That Sign Up requires `username` field

At the moment, Post author is `nil` and nothing is displayed herein; therefore, I can't give you a pass in this requirement because it isn't reflected in the UI, Sign Up form or Model validations, and there's no indication that this is a required attribute beyond being exclusively specified in here and in your `schema`.

As a signed-in user, I shall see the author of a Post [`REQUIRED`]

Reply:
I'm sorry, we didn't quite got this one! Could you help us to clarify if we fixed it properly?